### DISCUSSION TITLE
Quarto + Revealjs: How to change the logo size

### DISCUSSION THREAD

**Discussion by osdf** [quarto-dev/quarto-cli#7205] [Q&A] [answered]
### Bug description

There are some suggestions floating around on how to change the logo size on a revealjs slide (the logo is placed on the right side of the footer). These don't work however.

**Comment by mcanouil** [comment, marked as answer]
Very unclear what you are talking about with "There are some suggestions floating ... These don't work".
What does not work? What did you try?

Providing some example or even a reproducible example such as below (which works just fine) goes a long way:

```qmd
---
title: Quarto Playground
format: revealjs
logo: https://raw.githubusercontent.com/mcanouil/hex-stickers/main/SVG/mc.svg
include-in-header:
  - text: |
      <style>
      .reveal .slide-logo {
        max-height: unset;
        height: 100px;
      }
      </style>
---
```
